README.rst

```diff
@@ -40,6 +40,7 @@
 .. |SeabornMinVersion| replace:: 0.9.0
 .. |PytestMinVersion| replace:: 7.1.2
 .. |PlotlyMinVersion| replace:: 5.14.0
+.. |CythonMinVersion| replace:: 3.0.10
```

Comment:
I wouldn't include this one here. It's a build only dependency whereas these are more runtime + doc/examples dependencies.

Reply:
Then perhaps remove pytest from the list?

Reply:
Ah my bad these are just the substitution rules. We don't need cython because we're not referencing cython min version afterwards. Instead we're referencing pytest min version afterwards so we need to keep it